public/Create.js

```diff
@@ -54,6 +54,16 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
 //
```

Comment:
This file (along with the public/js/app.js) should be in the `.gitignore` file. It's best to exclude theentire  public JS and CSS directories from the git repo as they aren't required to be submitted to git

For example, add:
`public/js`
`public/css`